docs/content/tutorials/cluster.md

```diff
@@ -22,56 +22,109 @@ title: "Clustering"
   ~ under the License.
   -->
 
-# Clustering
+# Setting up a Clustered Deployment
 
 Apache Druid (incubating) is designed to be deployed as a scalable, fault-tolerant cluster.
 
 In this document, we'll set up a simple cluster and discuss how it can be further configured to meet
 your needs. 
 
 This simple cluster will feature:
- - A single Master server to host the Coordinator and Overlord processes
- - Scalable, fault-tolerant Data servers running Historical and MiddleManager processes
- - Query servers, hosting Druid Broker processes
+ - A Master server to host the Coordinator and Overlord processes
+ - Two scalable, fault-tolerant Data servers running Historical and MiddleManager processes
+ - A query server, hosting the Druid Broker and Router processes
 
-In production, we recommend deploying multiple Master servers with Coordinator and Overlord processes in a fault-tolerant configuration as well.
+In production, we recommend deploying multiple Master servers and multiple Query servers in a fault-tolerant configuration based on your specific fault-tolerance needs, but you can get started quickly with one Master and one Query server and add more servers later.
```

Comment:
What about migrating the metadata store from local derby to something else? I think we should talk about HA and the metadata store

Reply:
Nevermind, it is covered later